### PR TITLE
Undo breakage of multi-primary key entities with test extension

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Undo breakage of multi-primary key entities with test extension

Extension with basic test to replicate https://github.com/civicrm/civicrm-core/pull/24584 but adds 

Also - switching to master for this - that PR shoudl go in the rc & we can fiddle with the tests for master

### CODE REVIEW

File: tests/extensions/test.primary_keys/info.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  <classloader>
+    <psr4 prefix="Civi\" path="Civi"/>
+  </classloader>
+  <civix>
+    <namespace>CRM/PrimaryKeys</namespace>
+    <format>22.05.2</format>
+    <angularModule>crmPrimaryKeys</angularModule>
+  </civix>
+  <tags>
+    <tag>hidden</tag>
```
Comment (colemanw) [inline tests/extensions/test.primary_keys/info.xml:33]:
```suggestion
    <tag>mgmt:hidden</tag>
```

**Comment by eileenmcnaughton** [comment]
I committed the suggestion - but this extension probably needs to be fixed to run during tests if it is to add value - @seamuslee001 thoughts?

**Comment by eileenmcnaughton** [comment]
@seamuslee001 this extension just adds a test - if we merge it we have to configure it to run somehow.... (The thing the test was proving is fixed now)

**Comment by eileenmcnaughton** [comment]
@seamuslee001 should we merge or abandon this? The bug is fixed - this was the unit test but if we merge it then we need to also make jenkins run it or it is no use

**Comment by seamuslee001** [comment]
I'm leaning toward merging

**Comment by seamuslee001** [comment]
@eileenmcnaughton is there any reason why you didn't add this in the ext/ folder instead maybe with tag:hidden?

**Comment by eileenmcnaughton** [comment]
@seamuslee001 hmm I guess that could be done - it feels fugly to put test-only extensions there - I  thought there might be precedent - but I don't actually see any in there right now

**Comment by eileenmcnaughton** [comment]
This is what is in the `tests/extensions` folder right now

image

**Comment by eileenmcnaughton** [comment]
@seamuslee001 I'm gonna close this - I feel like it's probably better to call it quits on this one

**Pull request closed without merging**